charts/drupal/templates/backup-volume.yaml

```diff
@@ -16,16 +16,17 @@ spec:
   {{- if .Values.backup.csiDriverName }}
   csi:
     driver: {{ .Values.backup.csiDriverName }}
-    volumeHandle: {{ .Release.Namespace }}-{{ $.Release.Namespace | sha256sum | trunc 7 }}-backups
+    volumeHandle: {{ .Release.Namespace }}-{{ $.Release.Namespace | sha256sum | trunc 7 }}-backup
     volumeAttributes:
       remotePathSuffix: /{{ .Release.Namespace }}/{{ .Values.environmentName }}/backups
+      vfs-cache-mode: "off"
```

Comment:
So does this new line override [this one](https://github.com/wunderio/csi-rclone/blob/master/pkg/rclone/nodeserver.go#L247), or what is the current default?

Reply:
yes, it excludes flags when override is defined

https://github.com/wunderio/csi-rclone/blob/master/pkg/rclone/nodeserver.go#L294-L305